pynetdicom/transport.py

```diff
@@ -487,7 +487,7 @@ class AssociationServer(TCPServer):
         no TLS is required (default).
     """
     def __init__(self, ae, address, ae_title, contexts, ssl_context=None,
-                 evt_handlers=None):
+                 evt_handlers=None, RequestHandlerClass=None):
```

Comment:
Same

Reply:
FWIW, I made this match the `RequestHandlerClass` as used in the `socketserver` module: https://docs.python.org/3/library/socketserver.html#socketserver.TCPServer . 

I'll go ahead and change this, though.